executor/builder.go

```diff
@@ -784,11 +784,7 @@ func (b *executorBuilder) buildTopN(v *plan.PhysicalTopN) Executor {
 }
 
 func (b *executorBuilder) buildApply(apply *plan.PhysicalApply) *NestedLoopApplyExec {
-	v, ok := apply.PhysicalJoin.(*plan.PhysicalHashJoin)
```

Comment:
Why remove the type assert?

Reply:
The type of `PhysicalJoin` has been changed.